<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weather_forecast</class>
 <widget class="QWidget" name="weather_forecast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>城市</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back">
       <property name="text">
        <string>返回主页面</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,0);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>739</width>
        <height>854</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="10" column="2">
          <widget class="QLabel" name="img_15">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="img_8">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="img_26">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="13" column="4">
          <widget class="QLabel" name="img_32">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QLineEdit" name="weather_9"/>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="img_31">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QLineEdit" name="weather_11"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="img_0">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/new/prefix3/img/big_snow.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="5">
          <widget class="QLineEdit" name="T_13"/>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="img_41">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="day_8">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLineEdit" name="T_5"/>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="weather_5"/>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="img_3">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="day_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="T_1"/>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="img_21">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="day_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="13" column="3">
          <widget class="QLineEdit" name="weather_13"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="day_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="img_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="weather_0"/>
         </item>
         <item row="14" column="5">
          <widget class="QLineEdit" name="T_15"/>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="img_6">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="4">
          <widget class="QLabel" name="img_40">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="img_9">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QLineEdit" name="T_8"/>
         </item>
         <item row="6" column="5">
          <widget class="QLineEdit" name="T_7"/>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="T_3"/>
         </item>
         <item row="10" column="3">
          <widget class="QLineEdit" name="weather_10"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="img_23">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="day_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="img_10">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="day_4">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="img_22">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="weather_7"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="img_24">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QLabel" name="img_37">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="13" column="5">
          <widget class="QLineEdit" name="T_14"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="day_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="img_34">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="9" column="5">
          <widget class="QLineEdit" name="T_10"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="img_42">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="img_1">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="weather_1"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="img_16">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="weather_4"/>
         </item>
         <item row="13" column="2">
          <widget class="QLabel" name="img_29">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="weather_2"/>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="day_14">
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="14" column="4">
          <widget class="QLabel" name="img_44">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="9" column="4">
          <widget class="QLabel" name="img_43">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="weather_3"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="img_20">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="img_33">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="img_36">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="img_28">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="day_12">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="5">
          <widget class="QLineEdit" name="T_11"/>
         </item>
         <item row="8" column="5">
          <widget class="QLineEdit" name="T_9"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="day_10">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="img_2">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="5">
          <widget class="QLineEdit" name="T_12"/>
         </item>
         <item row="12" column="2">
          <widget class="QLabel" name="img_18">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="day_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="14" column="3">
          <widget class="QLineEdit" name="weather_14"/>
         </item>
         <item row="3" column="5">
          <widget class="QLineEdit" name="T_4"/>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="img_14">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="img_27">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="img_7">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="img_11">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="img_17">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="day_0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="img_30">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="day_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="img_19">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLabel" name="img_13">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="day_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="T_2"/>
         </item>
         <item row="10" column="4">
          <widget class="QLabel" name="img_35">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="11" column="4">
          <widget class="QLabel" name="img_39">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLineEdit" name="T_6"/>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="weather_6"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="day_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="img_4">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="img_12">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="2">
          <widget class="QLabel" name="img_25">
           <property name="text">
            <string>天气状况</string>
           </property>
          </widget>
         </item>
         <item row="12" column="3">
          <widget class="QLineEdit" name="weather_12"/>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="img_38">
           <property name="text">
            <string>温度</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLineEdit" name="weather_8"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
